modal/_utils/local_file_filter.py

```diff
@@ -0,0 +1,58 @@
+# Copyright Modal Labs 2024
+from pathlib import Path
+
+from .pattern_matcher import PatternMatcher
+
+
+class LocalFileFilter:
+    """Allows checking paths against one or more patterns."""
+
+    def __init__(self, *pattern: str, invert: bool = False) -> None:
```

Comment:
I'd prefer to not make invert an argument to the constructor, it can just be a lambda in the `__invert__` method since all we need is a callable :)